.idems_app/deployments/plh_asset_packs/config.ts

```diff
@@ -0,0 +1,12 @@
+import { extendDeploymentConfig } from "scripts";
+import { supabaseConfig } from "./encrypted/supabaseConfig";
```

Comment:
nit(non-blocking)

Really nice use of the encrypted configs, makes a lot of sense to pass the config into the deployment in this way, ensuring that the deployment will only compile if available.
It might also make sense to refactor the firebase config to behave in the same way (future issue)

The only issue with this is that we won't be able to swap to the deployment to decrypt the config initially, as you can only load a deployment that passes compilation. So we will need a workaround to either attempt decrypt before load, or make the import dynamic (e.g https://mariusschulz.com/blog/dynamic-import-expressions-in-typescript)

Can discuss on dev call